src/layouts/layout.css

```
@@ -1,7 +1,15 @@
+@import "tailwindcss";
+@plugin "@tailwindcss/typography";
```

Stylelint (17.5.0)

[error] Unexpected unknown at-rule "@plugin" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)